### PR TITLE
various roles: ansible-lint 6.22.0 cleanup

### PR DESCRIPTION
**Pull request by berndfinger**
Solves issue #535 for the following roles:
- sap_netweaver_preconfigure
- sap_hana_preconfigure
- sap_hana_install
- sap_swpm
- sap_install_media_detect
- sap_hostagent

### CODE REVIEW

File: roles/sap_hostagent/tasks/deploy_bundle.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 - name: Install SAPHOSTAGENT
-  command: "{{ sap_hostagent_agent_tmp_directory }}/global/hdb/saphostagent_setup/saphostexec -install"
+  ansible.builtin.command: "{{ sap_hostagent_agent_tmp_directory }}/global/hdb/saphostagent_setup/saphostexec -install"
   register: installagent
   args:
     chdir: "{{ sap_hostagent_agent_tmp_directory }}/global/hdb/saphostagent_setup/"
-  changed_when: "'Install service' in installagent.stdout"
\ No newline at end of file
+  changed_when: "'Install service' in installagent.stdout"
```
Comment (rhmk) [inline roles/sap_hostagent/tasks/deploy_bundle.yml:15]: Is the double quote irrelevant?

**Review by rhmk — APPROVED** [review]
LGTM

File: roles/sap_hostagent/tasks/deploy_bundle.yml (hunk 1 above)
Comment (berndfinger) [reply roles/sap_hostagent/tasks/deploy_bundle.yml:15]: Probably not. We have the same syntax in all other roles as per output of:
```
$ cd (collection root)
$ grep -rn "changed_when" | grep " in "
```
Also, while working on fixing https://github.com/sap-linuxlab/community.sap_install/issues/555, I got the following error message when removing the double quotes from the `that:` statement in one of the `assert` tasks:
```
  ansible.builtin.assert:
    that: 'bind-utils' in ansible_facts.packages
                       ^ here
This one looks easy to fix. It seems that there is a value started
with a quote, and the YAML parser is expecting to see the line ended
with the same kind of quote. For instance:

    when: "ok" in result.stdout

Could be written as:

   when: '"ok" in result.stdout'

Or equivalently:

   when: "'ok' in result.stdout"
```

Comment (rhmk) [reply roles/sap_hostagent/tasks/deploy_bundle.yml:15]: Thanks for the explanation.

**Pull request merged**